lib/opencog.conf

```diff
@@ -107,9 +107,9 @@ PYTHON_PRELOAD = pyshell, restapi, pattern_match_functions, agent_finder
 # For information on how to set up atomspace persistance, see the README
 # file in opencog/persist/sql/README.
 #
-# STORAGE               = "opencog-atomspace"
-# STORAGE_USERNAME      = "opencog_user"
-# STORAGE_PASSWD        = "asdf"
+STORAGE               = "OpenCogTut"
+STORAGE_USERNAME      = "rohits134"
+STORAGE_PASSWD        = "rohits134"
 #
```

Comment:
This looks specific to your local computer's configuration, so perhaps it shouldn't be part of the pull request?